Makefile

```diff
@@ -527,4 +527,4 @@ quick-deploy-hubble:
 
 .PHONY: simplify-dashboards
 simplify-dashboards:
-	cd deploy/legacy/grafana/dashboards/ && go test . -tags=dashboard,simplifydashboard -v
+	go test -tags=dashboard,simplifydashboard -v ./deploy/...
```

Comment:
The test logic actually modifies JSON files in the current directory. Could we cd into both directories?